Makefile

```diff
@@ -94,10 +94,16 @@ put-translations:
 linguas:
 	@${SRC_DIR}/tools/make-LINGUAS.sh 80 > ${SRC_DIR}/locale/LINGUAS
 
+lint: lint-js lint-css
+
 lint-js:
 	cd ${JS_DIR} \
 	&& npm run lint
 
+lint-css:
+	cd ${JS_DIR} \
+	&& npm run stylelint -- --config=stylelint.json "../css/**/*.css"
```

Comment:
As I understand this will only check for CSS files under the _static/css/_ folder, but not CSS files which live next to components and we'll want to check too.